infrastructure/cli/auth/initializer.go

```diff
@@ -59,8 +59,10 @@ func (i *Initializer) Init() error {
 		return nil
 	}
 	if !currentConfig.AutomaticAuthentication() {
-		notification.Send(sglsp.ShowMessageParams{Message: "Please authenticate your Snyk user in order to start scanning"})
-		log.Info().Msg("Skipping authentication - automatic authentication is disabled")
+		if currentConfig.Token() != "" { // Only send notification when the token is invalid
+			notification.SendError(&AuthenticationFailedError{})
```

Comment:
This will print `You can reset the token to re-authenticate automatically` in notification. That is wrong, since it's non-automatic auth case.